RELEASE/scripts/autoscend/auto_cooking.ash

```diff
@@ -611,6 +606,52 @@ void consumeStuff()
 	}
 }
 
+boolean consumeFortune()
+{
+	if (contains_text(get_counters("Fortune Cookie", 0, 200), "Fortune Cookie"))
+	{
+		return false;
+	}
+
+	// Don't get lucky numbers for the first semi-rare if we still need to adventure in the outskirts
+	if (my_turncount() < 80 && internalQuestStatus("questL05Goblin") == 0)
+	{
+		return false;
+	}
+
+	if (my_level() > 12)
+	{
+		return false;
+	}
+
+	// Try to consume a Lucky Lindy
+	if (inebriety_left() > 0 && auto_is_valid($item[Lucky Lindy]) && my_meat() >= npc_price($item[Lucky Lindy]))
+	{
+		if (autoDrink(1, $item[Lucky Lindy]))
+		{
+			return true;
+		}
+	}
+	
+	// Try to consume a Fortune Cookie
+	if (fullness_left() > 0 && my_meat() >= npc_price($item[Fortune Cookie]))
+	{
+		// Don't eat a fortune cookie if a spaghetti breakfast is still consumable
+		if (auto_is_valid($item[Spaghetti Breakfast]) && my_fullness() == 0 && my_level() >= 10)
```

Comment:
```suggestion
		if (auto_is_valid($item[Spaghetti Breakfast]) && have_skill($skill[Spaghetti Breakfast]) && !get_property("_spaghettiBreakfastEaten").to_boolean() && my_fullness() == 0 && my_level() >= 10)
```

Otherwise this screws everyone who doesn't have the skill out of consuming Fortune Cookies when they're level 10 or 11.

Reply:
i should probably check available_amount instead